README.md

```diff
@@ -21,6 +24,20 @@ Verify that your application uses DXVK instead of wined3d by checking for the pr
 
 In order to remove DXVK from a prefix, remove the DLLs and DLL overrides, and run `wineboot -u` to restore the original DLL files.
 
+### External tools
+
+On Linux there exists a wide ecosystem of programs to manage your Wine game installs. These tools can also install or even auto-update DXVK. To name a few tools with different capabilities
+[Bottles](https://github.com/bottlesdevs/Bottles), 
+[Heroic](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher), 
+[Lutris](https://github.com/lutris/lutris) and [Winetricks](https://github.com/Winetricks/winetricks). There is also a simple [setup_script.sh](https://gist.github.com/doitsujin/1652e0e3382f0e0ff611e70142684d01) that might work for your use case. This is clearly not an exhaustive list nor a recommendation to use any of said tools. Use the method that you best see fit!
```

Comment:
Nice! Tho, it's not really clear what's the difference between the 3 earlier tools and the script *(and they are clearly completely different)*. How about changing the sentence

> There is also a simple [setup_script.sh](https://gist.github.com/doitsujin/1652e0e3382f0e0ff611e70142684d01) that might work for your use case

to

> There is also a simple [setup_script.sh](https://gist.github.com/doitsujin/1652e0e3382f0e0ff611e70142684d01), purposed specifically to install/remove DXVK from a WINEPREFIX, that might work for your use case

Reply:
*(I'm not a native english speaker btw, so pardon me if that may be further improved)*

Reply:
yeah :+1: